examples/dmd/dg_euler.ipynb

```diff
@@ -0,0 +1,2400 @@
+    "                    action='store', default=0, type=int,\n",
+    "                    help=\"Number of times to refine the mesh uniformly before parallel.\")\n",
+    "parser.add_argument('-rp', '--refine_parallel',\n",
+    "                    action='store', default=1, type=int,\n",
+    "                    help=\"Number of times to refine the mesh uniformly after parallel.\")\n",
+    "parser.add_argument('-o', '--order',\n",
+    "                    action='store', default=3, type=int,\n",
+    "                    help=\"Finite element order (polynomial degree)\")\n",
+    "parser.add_argument('-s', '--ode_solver',\n",
+    "                    action='store', default=4, type=int,\n",
+    "                    help=\"ODE solver: 1 - Forward Euler,\\n\\t\" +\n",
+    "                    \"            2 - RK2 SSP, 3 - RK3 SSP, 4 - RK4, 6 - RK6.\")\n",
+    "parser.add_argument('-tf', '--t_final',\n",
+    "                    action='store', default=2.0, type=float,\n",
+    "                    help=\"Final time; start time is 0.\")\n",
+    "parser.add_argument(\"-dt\", \"--time_step\",\n",
+    "                    action='store', default=-0.01, type=float,\n",
+    "                    help=\"Time step.\")\n",
+    "parser.add_argument('-c', '--cfl_number',\n",
+    "                    action='store', default=0.3, type=float,\n",
+    "                    help=\"CFL number for timestep calculation.\")\n",
+    "parser.add_argument('-vis', '--visualization',\n",
+    "                    action='store_true',\n",
+    "                    help='Enable GLVis visualization')\n",
+    "parser.add_argument('-visit', '--visit-datafiles',\n",
+    "                    action='store_true', default=False,\n",
+    "                    help='Save data files for VisIt (visit.llnl.gov) visualize.')\n",
+    "parser.add_argument('-vs', '--visualization_steps',\n",
+    "                    action='store', default=50, type=float,\n",
+    "                    help=\"Visualize every n-th timestep.\")\n",
+    "\n",
+    "# additional args for DMD\n",
+    "parser.add_argument('-ef','--energy_fraction',\n",
+    "                    action='store',default=0.9999, type=float,\n",
+    "                    help='Energy fraction for DMD')\n",
+    "parser.add_argument('-rdim','--rdim',\n",
+    "                    action='store',default=-1,type=int,\n",
+    "                    help='Reduced dimension for DMD')\n",
+    "parser.add_argument('-crbf','--crbf',\n",
+    "                    action='store',default=0.9,type=float,\n",
+    "                    help='Closest RBF value')\n",
+    "parser.add_argument('-nonunif','--nonunif',dest='nonunif',\n",
+    "                    action='store_true',help='Use NonuniformDMD')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "05c0a673-0043-4696-8ec2-7c88f456fc64",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "args = parser.parse_args([])\n",
```

Comment:
It might be helpful to add some comments to each example showing how to pass the command line arguments for the different sample runs.

For example, the first sample run for the DG_Euler [example here](https://github.com/LLNL/pylibROM/blob/ee7a47ecb518153d02a771c7eadfb40c62f30375/examples/dmd/dg_euler.py#L24) can be run in the Jupyter notebook with ```args = parser.parse_args("-p 1 -rs 1 -rp 1 -o 5 -s 6 -tf 0.1".split())```